### PR TITLE
Added support for all RPC commands inside the websocket

### PR DESCRIPTION
**Pull request by MarcoOl94**

### CODE REVIEW

File: sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import io.horizen.account.api.rpc.response.RpcResponseError
+import io.horizen.account.api.rpc.utils.{RpcCode, RpcError}
+import io.horizen.account.serialization.EthJsonMapper
+import sparkz.util.SparkzLogging
+
+import scala.jdk.CollectionConverters.asScalaIteratorConverter
+import scala.util.{Failure, Success, Try}
+
+
+object RpcProcessor extends SparkzLogging {
```
Comment (i-Alex) [inline sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala:15]: I still vote to keep `RpcProcessor` as a class, not as an object.
In the AccountSidechainApp create an instance of this class and then pass as an argument of:
1) `AccountEthRpcRoute` constructor, to let this class use RpcProcessor explicitly. Implicit dependencies are not good to have.
2) `WebSocketAccountServerRef` constructor, and then inside do the same trick as for NVHactor.

File: sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      EthJsonMapper.serialize(responses)
+    } else {
+      EthJsonMapper.serialize(responses.head)
+    }
+
+    log.trace(s"RPC message response << $json")
+    json
+  }
+
+  def getClientVersion: String = {
```
Comment (i-Alex) [inline sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala:51]: do we really need to have this method here?
Let's move it to a separate utility class inside `package io.horizen.account.api.rpc.service`

**Review by i-Alex — CHANGES_REQUESTED** [review]

File: sdk/src/main/scala/io/horizen/account/websocket/WebSocketAccountServerEndpoint.scala [hunk 3]
```diff
@@ -307,4 +309,13 @@ private object WebSocketAccountServerEndpoint extends SparkzLogging {
     })
   }
 
+  def sendRpc(rpcResponse: Object, session: Session): Unit = {
```
[the remark this answers is not in the document]
Comment (i-Alex) [reply sdk/src/main/scala/io/horizen/account/websocket/WebSocketAccountServerEndpoint.scala:312]: beside that, let's extend py tests

[the remark this answers is not in the document]
Comment (MarcoOl94) [reply sdk/src/main/scala/io/horizen/account/websocket/WebSocketAccountServerEndpoint.scala]: Done

File: sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala (hunk 1 above)
Comment (MarcoOl94) [reply sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala]: Done

File: sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala (hunk 2 above)
Comment (MarcoOl94) [reply sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala]: Done

**Review by i-Alex — APPROVED** [review]

**Pull request merged**